src/Behat/Gherkin/Node/FeatureNode.php

```diff
@@ -78,6 +78,10 @@ public function __construct(
         $file,
         $line
     ) {
+        // Verify that the feature file is an absolute path.
+        if (!empty($file) && (!is_string($file) || $file[0] !== DIRECTORY_SEPARATOR)) {
```

Comment:
On windows, the first char is not `DIRECTORY}_SEPARATOR`...

Reply:
How about using ``file_exists(realpath($file))``? Throw an exception if it fails. If not keep using the new generated absolute path.